Swivl/Sniffs/Commenting/DoctrineEntitySniff.php

```diff
@@ -934,7 +940,11 @@ protected function validateMethodDeclaration(
             if ($argumentName !== null) {
                 $argsOpenPtr = $tokens[$methodPtr]['parenthesis_opener'];
                 $argsClosePtr = $tokens[$methodPtr]['parenthesis_closer'];
-                $argPtr = $this->phpcsFile->findNext([T_NS_SEPARATOR, T_NULLABLE, T_STRING, T_VARIABLE], $argsOpenPtr, $argsClosePtr);
+                $argPtr = $this->phpcsFile->findNext(
```

Comment:
Please revert all changes in codestyle, because we should configure travis for checking it before fixing. Make separate PR to fix codestyle.